web/src/components/RunCard/TestSuiteRunCard.tsx

```diff
@@ -15,68 +15,71 @@ interface IProps {
 }
 
 const TestSuiteRunCard = ({
-  run: {id: runId, createdAt, state, metadata, version, pass, fail, allStepsRequiredGatesPassed},
+  run: {
+    id: runId,
+    createdAt,
+    state,
+    metadata: {name, buildNumber, branch, url, source},
+    version,
+    pass,
+    fail,
+    allStepsRequiredGatesPassed,
+  },
   testSuiteId,
   linkTo,
-}: IProps) => {
-  const metadataName = metadata?.name;
-  const metadataBuildNumber = metadata?.buildNumber;
-  const metadataBranch = metadata?.branch;
-  const metadataUrl = metadata?.url;
-
-  return (
-    <Link to={linkTo}>
-      <S.Container $isWhite>
-        <TestSuiteRunStatusIcon state={state} hasFailedTests={!allStepsRequiredGatesPassed} />
-        <S.Info>
-          <div>
-            <S.Title>v{version}</S.Title>
-          </div>
-          <S.Row>
-            <Tooltip title={Date.format(createdAt)}>
-              <S.Text>{Date.getTimeAgo(createdAt)}</S.Text>
-            </Tooltip>
-            {/* Adding this latter when is available */}
-            {/* <S.Text>&nbsp;• 0s (executionTime missing from API)</S.Text> */}
-
-            {metadataName && (
-              <a href={metadataUrl} target="_blank" onClick={event => event.stopPropagation()}>
-                <S.Text $hasLink={Boolean(metadataUrl)}>&nbsp;• {`${metadataName} ${metadataBuildNumber}`}</S.Text>
-              </a>
-            )}
-            {metadataBranch && <S.Text>&nbsp;• Branch: {metadataBranch}</S.Text>}
-          </S.Row>
-        </S.Info>
-
-        {state !== TestStateEnum.FAILED && state !== TestStateEnum.FINISHED && (
-          <div>
-            <TestState testState={state} />
-          </div>
-        )}
+}: IProps) => (
+  <Link to={linkTo}>
+    <S.Container $isWhite>
+      <TestSuiteRunStatusIcon state={state} hasFailedTests={!allStepsRequiredGatesPassed} />
+      <S.Info>
+        <div>
+          <S.Title>v{version}</S.Title>
+        </div>
+        <S.Row>
+          <Tooltip title={Date.format(createdAt)}>
+            <S.Text>{Date.getTimeAgo(createdAt)}</S.Text>
+          </Tooltip>
+          {source && <S.Text>&nbsp;• Run via {source.toUpperCase()}</S.Text>}
+          {/* Adding this latter when is available */}
+          {/* <S.Text>&nbsp;• 0s (executionTime missing from API)</S.Text> */}
```

Comment:
Is this a dead code? Or is it something that will be added in future PRs?

Reply:
I think we should go back and add this to the BE, iw would be nice to have this in the UI